anthemav/protocol.py

```diff
@@ -394,7 +422,10 @@ async def _assemble_buffer(self):
         disassembles the chain of datagrams into individual messages which
         are then passed on for interpretation.
         """
-        self.transport.pause_reading()
+        try:
+            self.transport.pause_reading()
+        except AttributeError:
+            self.log.warning('Lost connection to receiver while assembling buffer')
```

Comment:
What the benefit of adding this? Does this help the protocol to reconnect when connection is lost. Which would potentially fix this bug: https://github.com/nugget/python-anthemav/issues/42

Reply:
I forgot about that.  It could be spun off to a separate PR if you prefer.

I don't remember the exact details but what I found was that if the connection was lost for any reason - usually because I used the Anthem app to control the device, thus kicking out this library's connection -  the `pause_reading()` call would throw an exception and every subsequent operation would fail, spamming my logs with further errors.

This block just catches the exception and allows us to reconnect later.  So yes, it might well fix #42.